server/controllers/reviews.js

```diff
@@ -0,0 +1,25 @@
+const Reviews = require('../models/Reviews');
+const asyncHandler = require('express-async-handler');
+
+exports.getAllReviews = asyncHandler(async (req, res) => {
+  try {
+    const info = await Reviews.find({
+      userId: "60ef477aa875e15d00c7c51e"
+    });
+
+    res.status(200).json(info);
+  } catch (err) {
+    res.status(500).json(err);
+  }
+});
+
+exports.createReview = asyncHandler(async (req, res) => {
+  try {
+    const review = await Reviews.create({
+      ...req.body
```

Comment:
Are we receiving the user's id from the request body here? (Maybe you can get it from the `req.user` object instead of sending it through the body)